tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1640,6 +1693,10 @@ int tester_gap_padv_create_sync(struct bt_le_per_adv_sync_param *create_params)
 
 int tester_gap_padv_stop_sync(void)
 {
+	if (!IS_ENABLED(CONFIG_BT_PER_ADV)) {
```

Comment:
```suggestion
	if (!IS_ENABLED(CONFIG_BT_PER_ADV_SYNC)) {
```

Reply:
Would it make sense to add proposed options to handlers registration and btp_gap header too?

Reply:
It's a bit curious, because e.g. `tester_gap_padv_stop_sync` is only called in a BAP Broadcast Sink context, which requires `CONFIG_BT_PER_ADV_SYNC` already, so it doesn't really make sense to guard the function call in those contexts at least.